Linting is failing on index.ts

As of today, we're hitting lint errors when building...
```
yarn run eslint
yarn run v1.17.3
$ eslint . --fix --ignore-path .gitignore --ext .ts,.tsx

/Users/kbates/repos/oss/elyra-ai/elyra/packages/application/src/index.ts
  0:0  error  Parsing error: Cannot read property 'name' of undefined

✖ 1 problem (1 error, 0 warnings)
```

--- a/package.json
+++ b/package.json
@@ -19,8 +19,8 @@
     }
   },
   "devDependencies": {
-    "@typescript-eslint/eslint-plugin": "^2.14.0",
-    "@typescript-eslint/parser": "^2.14.0",
+    "@typescript-eslint/eslint-plugin": "~2.23.0",
+    "@typescript-eslint/parser": "~2.23.0",
     "eslint": "^6.5.0",
     "eslint-config-prettier": "^6.9.0",
     "eslint-plugin-header": "^3.0.0",

